modules/customize-installation-operator-overlays.adoc

```diff
@@ -39,21 +39,21 @@ The following example shows the structure of an overlay:
 [source,yaml]
 ----
 overlays:
-- apiVersion: v1     # <1>
-  kind: ConfigMap    # <2>
-  name: my-configmap # <3>
+- apiVersion: v1         # <1>
+  kind: ExampleKind      # <2>
+  name: my-resource      # <3>
   patches:
-    - path: .data    # <4>
-      value: |       # <5>
+    - path: .some.field  # <4>
+      value: |           # <5>
         key1: data2
         key2: data2
 ----
 
 <1> Targeted Kubernetes resource ApiVersion, for example `apps/v1`, `v1`, `networking.k8s.io/v1`
 <2> Resource type (e.g., Deployment, ConfigMap, NetworkPolicy)
-<3> Name of the resource, for example `my-configmap`
+<3> Name of the resource, for example `my-resource`.
```

Comment:
```suggestion
<3> Name of the resource, for example `my-resource`
```
Sorry, I noticed in the preview (which wasn't displaying correctly before) that all the other items don't have punctuation (except #5) so can you revert this change? Sorry!

Reply:
(since # 5 is two sentences it needs that period. the others don't)

Reply:
Done.